go-apps/meep-wais/server/wais.go

```diff
@@ -513,6 +622,19 @@ func checkAssocStaNotificationRegisteredSubscriptions(staMacIds []string, apMacI
 						}
 					default:
 					}
+
+					//if the notification already triggered, do not send it again unless its a periodic event
+					if match {
+						if !isPeriodicInvoked {
+							if !subInfo.Triggered {
+								assocStaSubscriptionInfoMap[subsId].Triggered = true
+							} else {
+								match = false
```

Comment:
Same comment about debugging. In this case "match" is set to false even when there is a match, correct? It may be clearer to use another flag here to indicate that periodic is enabled but should not be triggered until the timer expires.

Reply:
changed to give it a clearer name using another flag within the for loop.. but essentially same logic